pkg/kubelet/kuberuntime/kuberuntime_sandbox_linux.go

```diff
@@ -55,7 +55,11 @@ func (m *kubeGenericRuntimeManager) calculateSandboxResources(pod *v1.Pod) *runt
 	if _, cpuRequestExists := req[v1.ResourceCPU]; cpuRequestExists {
 		cpuRequest = req.Cpu()
 	}
-	return m.calculateLinuxResources(cpuRequest, lim.Cpu(), lim.Memory())
+
+	// If pod has exclusive cpu the sandbox will not have cfs quote enforced
+	disableCPUQuota := utilfeature.DefaultFeatureGate.Enabled(features.DisableCPUQuotaWithExclusiveCPUs) && m.containerManager.PodHasExclusiveCPUs(pod)
```

Comment:
let's add a log entry here

Reply:
added

Reply:
we can use a common format in all the places
```
klog.V(2).InfoS("Disabled CFS quota", "pod", klog.KObj(pod))
```

Reply:
added same format across all!